vizro-ai/pyproject.toml

```diff
@@ -15,7 +15,7 @@ classifiers = [
 dependencies = [
   "pandas",
   "tabulate",
-  "openai>=0.27.8",
+  "openai>=0.27.8,<=0.28.1",  # TODO add support for openai>=1.0.0
```

Comment:
Should we not pin the to `<0.1.0`, so that we get e.g. `0.28.2` (if they ever release it)

Reply:
Fixed